biojava-structure-gui/src/main/java/org/biojava/nbio/structure/align/gui/DBResultTable.java

```diff
@@ -143,11 +146,11 @@ else if ( str.startsWith("#param:scoring=")){
 							}
 						}
 					} catch (IndexOutOfBoundsException e){
-						System.err.println("Unknown scoring strategy from line: " + str);
+						LOGGER.error("Unknown scoring strategy from line: " + str);
 					} catch (IllegalArgumentException e) {
-						System.err.println("Unknown scoring strategy from line: " + str);
+						LOGGER.error("Unknown scoring strategy from line: " + str);
 					} catch (Exception e) {
-						System.err.println("Unknown parameter can't read parameters from line: " + str);
+						LOGGER.error("Unknown parameter can't read parameters from line: " + str);
 						e.printStackTrace();
```

Comment:
Yea,I know that, but this addresses another rule here: https://dev.eclipse.org/sonar/coding_rules#q=squid%3AS1148

Will fix the occurrences of this rule in another PR